extension/config/src/main/resources/conf/platform/exo-sample.properties

```diff
@@ -692,6 +692,23 @@
 
 ###########################
 #
+# social
+#
+
+# To disable any Activity Type add exo.activity-type.<activity-type-key>.enabled=false
+# by default all activity Type are activated.
+# The list of all activity type key {DEFAULT_ACTIVITY,SPACE_ACTIVITY,USER_ACTIVITIES_FOR_SPACE,exosocial:relationship,
+# LINK_ACTIVITY,sharecontents:spaces,USER_PROFILE_ACTIVITY,DOC_ACTIVITY,files:spaces,sharefiles:spaces,contents:spaces,
+# cs-calendar:spaces,TaskAdded,ks-forum:spaces,ks-answer:spaces,ks-poll:spaces,ks-wiki:spaces,USER_ACTIVITIES_FOR_RELATIONSHIP}
+#
+# exo.activity-type.DOC_ACTIVITY.enabled=true
+# exo.activity-type.cs-calendar:spaces.enabled=true
```

Comment:
List all activity types in comment, for example:

+# exo.activity-type.DOC_ACTIVITY.enabled=true
 +# exo.activity-type.cs-calendar:spaces.enabled=true
 +# exo.activity-type.ks-poll:spaces.enabled=true
 +# exo.activity-type.USER_ACTIVITIES_FOR_RELATIONSHIP.enabled=true
...